bin/replication.js

```diff
@@ -2,10 +2,14 @@ const async = require('async');
 const program = require('commander');
 const { S3, IAM, SharedIniFileCredentials } = require('aws-sdk');
 
-const { Logger } = require('werelogs');
-
+const werelogs = require('werelogs');
+const { RoundRobin } = require('arsenal').network;
+const Logger = werelogs.Logger;
```

Comment:
It's because I wanted to configure Werelogs (the new way) log levels.
```
werelogs.configure({
    level: config.log.logLevel,
     dump: config.log.dumpLevel,
 });
```